src/index.spec.js

```diff
@@ -14,7 +14,6 @@ describe('babel plugin', () => {
   //       to starting subprocesses.
   var procPool
   beforeEach(() => { procPool = new ProcessPool })
-  afterEach(() => { procPool.destroy() })
```

Comment:
I just tried it with node 5.10.1, could you rebase to the latest version and try again without your process-leaking change to the tests?

If it still doesn't work... could you still remove the change from this PR so I can accept the babel opts fix. We can look at the tests not working on your system in a separate github issue.